test/agentchat/test_function_call.py

```diff
@@ -238,7 +238,7 @@ def test_update_function():
     config_list_gpt4 = autogen.config_list_from_json(
         OAI_CONFIG_LIST,
         filter_dict={
-            "tags": ["gpt-4", "gpt-4-32k", "gpt-4o", "gpt-4o-mini"],
+            "tags": ["gpt-4", "gpt-4-32k", "gpt-4o", "gpt-4o-mini", "o1-preview", "o1-mini"],
```

Comment:
Function call is not supported in o1. Do we need to change this?

Reply:
Good point, I'll remove.